src/RIEStatefulBase.js

```diff
@@ -55,7 +55,13 @@ export default class RIEStatefulBase extends RIEBase {
             {...this.props.editProps} />;
     };
 
-    renderNormalComponent = () => {
+    renderEditingComponent = () => {
+        return this.props.renderEditingComponent
+          ? this.props.renderEditingComponent.call(this)
```

Comment:
So I am checking here if I have the `renderEditingComponent` defined and the I want to call this function with the context of `current` object. 

The problem lays in the `.finishEditing` function, where we base the value from `.refs.value` and the `refs` looks like it is undefined. 

Should this happen? What am I missing?